### PR TITLE
Adding test for new sequence mode

### PR DESCRIPTION
**Pull request by debermudez**
Adding new test to cover new sequence mode

### CODE REVIEW

File: qa/L0_perf_analyzer/test.sh [hunk 1]
```diff
@@ -854,7 +854,34 @@ if [ $(cat $CLIENT_LOG |  grep "Request Rate: 40" | wc -l) -eq 0 ]; then
 fi
 set -e
 
-
+# Test --serial-sequences mode 
+set +e
```
Comment (tgerdesnv) [inline qa/L0_perf_analyzer/test.sh:858]: one set +e, two set -e?

File: qa/L0_perf_analyzer/test.sh [hunk 2]
```diff
@@ -854,7 +854,34 @@ if [ $(cat $CLIENT_LOG |  grep "Request Rate: 40" | wc -l) -eq 0 ]; then
...
+        echo -e "\n***\n*** Test Failed\n***"
+        RET=1
+    fi
+    if [ $(cat $CLIENT_LOG |  grep "${ERROR_STRING}" | wc -l) -ne 0 ]; then
+        cat $CLIENT_LOG
+        echo -e "\n***\n*** Test Failed\n***"
+        RET=1
+    fi
+    set -e
+$PERF_ANALYZER -v -i $PROTOCOL -m  simple_savedmodel_sequence_object -p 1000 --request-intervals $CUSTOM_SCHEDULE_FILE --serial-sequences \
```
Comment (tgerdesnv) [inline qa/L0_perf_analyzer/test.sh:872]: add endline between

File: qa/L0_perf_analyzer/test.sh [hunk 3]
```diff
@@ -854,7 +854,34 @@ if [ $(cat $CLIENT_LOG |  grep "Request Rate: 40" | wc -l) -eq 0 ]; then
 fi
 set -e
 
-
+# Test --serial-sequences mode 
+set +e
+$PERF_ANALYZER -v -i $PROTOCOL -m  simple_savedmodel_sequence_object -p 1000 --request-rate-range 100:200:50 --serial-sequences \
+    --input-data=$SEQ_JSONDATAFILE -s ${STABILITY_THRESHOLD} >$CLIENT_LOG 2>&1
+    if [ $? -ne 0 ]; then
```
Comment (tgerdesnv) [inline qa/L0_perf_analyzer/test.sh:861]: why indented?

Comment (debermudez) [reply qa/L0_perf_analyzer/test.sh]: un indented.

**Review by tgerdesnv — APPROVED** [review]

**Pull request merged**